Libraries/Components/Button.windows.js

```diff
@@ -116,6 +143,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    var content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      var tabIndex = this.props.tabIndex || 0;
+      var windowsTabFocusable = !disabled && tabIndex >= 0;
+      content =
+        <FocusableWindows
+          ref="focusable"
+          disabled={disabled}
+          isTabStop={windowsTabFocusable}
+          tabIndex={tabIndex}
+          disableSystemFocusVisuals={this.props.disableSystemFocusVisuals}
+          handledKeyDownKeys={DOWN_KEYCODES}
+          handledKeyUpKeys={UP_KEYCODES}
+          onKeyDown={this._onKeyDown}
+          onKeyUp={this._onKeyUp}
+          onFocus = {this._onFocus}
```

Comment:
>  [](start = 17, length = 1)

remove extra spacing around `=` for `onFocus` and `onBlur`

Reply:
fixed.

---
In reply to: [156557315](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156557315) [](ancestors = 156557315)